doc/oper.xml

```diff
@@ -1387,3 +1387,34 @@ gap> DigraphShortestDistance(gr, [1, 3], [3, 5]);
   </Description>
 </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="PartialOrderDigraphMeetOfVertices">
+<ManSection>
+  <Oper Name="PartialOrderDigraphMeetOfVertices" Label="for a digraph and two vertices"
+        Arg="digraph, u, v"/>
+  <Oper Name="PartialOrderDigraphJoinOfVertices" Label="for a digraph and two vertices"
+        Arg="digraph, u, v"/>
+  <Returns>An integer</Returns>
```

Comment:
I think we decided the following functionality for these operations:

If the digraph is not a partial order digraph, then do an `ErrorNoReturn`. 
Otherwise, if the meet/join does not exist, then return `fail`.

The first is an error because it doesn't make sense to ask for meets and joins in a digraph that isn't a partial order. And the second is fail because it's a valid question to ask, it's just that the things don't exist.

Therefore the implementation, and the documentation, should reflect this functionality.